### DISCUSSION TITLE
Experiencing strangeness with images

### DISCUSSION THREAD

**Discussion by wbconnor** [davidmyersdev/octo#206] [General]
I had a couple strange experiences with images recently:

- Wasn't working when trying to drag & drop into Octo, but copying & pasting worked fine
- When reopening the document the next day, the link for the image was there, but the image didn't show up until I started editing the document.

**Comment by davidmyersdev** [comment]
It's hard to track down these types of errors without logs or reproducible steps. I think a good start on this might actually be some tooling that allows you to export your current settings and _possibly_ error logs.
